daserste.de throws 404

### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.02.03**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Germany

### Description

Downloading from https://www.daserste.de/information/politik-weltgeschehen/weltspiegel/sendung/paraguay-deutsche-impf-gegner-100.html fails with a 404.

### Verbose log

```shell
> yt-dlp -vU "https://www.daserste.de/information/politik-weltgeschehen/weltspiegel/sendung/paraguay-deutsche-impf-gegner-100.html"
[debug] Command-line config: ['-vU', 'https://www.daserste.de/information/politik-weltgeschehen/weltspiegel/sendung/paraguay-deutsche-impf-gegner-100.html']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, err UTF-8, pref UTF-8
[debug] yt-dlp version 2022.02.03 [28469edd7]
[debug] Python version 3.6.15 (CPython 64bit) - Linux-5.3.18-59.40-default-x86_64-with-glibc2.3.4
[debug] exe versions: ffmpeg 3.4.9, ffprobe 3.4.9, rtmpdump 2.4
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
Latest version: 2022.02.03, Current version: 2022.02.03
yt-dlp is up to date (2022.02.03)
[debug] [ARD] Extracting URL: https://www.daserste.de/information/politik-weltgeschehen/weltspiegel/sendung/paraguay-deutsche-impf-gegner-100.html
[ARD] paraguay-deutsche-impf-gegner-100: Downloading XML
ERROR: [ARD] paraguay-deutsche-impf-gegner-100: Unable to download XML: HTTP Error 404: Not Found (caused by <HTTPError 404: 'Not Found'>); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using -U (caused by <HTTPError 404: 'Not Found'>); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using -U
  File "/usr/lib/python3.6/site-packages/yt_dlp/extractor/common.py", line 730, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/usr/lib/python3.6/site-packages/yt_dlp/YoutubeDL.py", line 3558, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib64/python3.6/urllib/request.py", line 532, in open
    response = meth(req, response)
  File "/usr/lib64/python3.6/urllib/request.py", line 642, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib64/python3.6/urllib/request.py", line 570, in error
    return self._call_chain(*args)
  File "/usr/lib64/python3.6/urllib/request.py", line 504, in _call_chain
    result = func(*args)
  File "/usr/lib64/python3.6/urllib/request.py", line 650, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
```

Hi,

I tried your URL too and I get first this error info....
```
[debug] Command-line config: ['https://www.daserste.de/information/politik-weltgeschehen/weltspiegel/sendung/paraguay-deutsche-impf-gegner-100.html', '-v', '-o', '-']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, err utf-8, pref cp1252
[debug] yt-dlp version 2022.02.03 [28469ed] (win_exe)
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.19043-SP0
[debug] exe versions: ffmpeg 4.0.1, ffprobe 4.0.1, rtmpdump 2.4
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
[debug] [ARD] Extracting URL: https://www.daserste.de/information/politik-weltgeschehen/weltspiegel/sendung/paraguay-deutsche-impf-gegner-100.html
[ARD] paraguay-deutsche-impf-gegner-100: Downloading XML
←[0;31mERROR:←[0m [ARD] paraguay-deutsche-impf-gegner-100: Unable to download XML: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)> (caused by URLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)'))) (caused by URLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))
  File "yt_dlp\extractor\common.py", line 730, in _request_webpage
  File "yt_dlp\YoutubeDL.py", line 3558, in urlopen
  File "urllib\request.py", line 525, in open
  File "urllib\request.py", line 542, in _open
  File "urllib\request.py", line 502, in _call_chain
  File "yt_dlp\utils.py", line 1501, in https_open
  File "urllib\request.py", line 1357, in do_open
```
...much certificate errors.I get this certificate error often and then I need to use this command...
`--no-check-certificates`
...to bypass the certificate problem.Otherwise I get the same error as you...

```
[debug] Command-line config: ['https://www.daserste.de/information/politik-weltgeschehen/weltspiegel/sendung/paraguay-deutsche-impf-gegner-100.html', '-v', '--no-check-certificates', '-o', '-']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, err utf-8, pref cp1252
[debug] yt-dlp version 2022.02.03 [28469ed] (win_exe)
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.19043-SP0
[debug] exe versions: ffmpeg 4.0.1, ffprobe 4.0.1, rtmpdump 2.4
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
[debug] [ARD] Extracting URL: https://www.daserste.de/information/politik-weltgeschehen/weltspiegel/sendung/paraguay-deutsche-impf-gegner-100.html
[ARD] paraguay-deutsche-impf-gegner-100: Downloading XML
←[0;31mERROR:←[0m [ARD] paraguay-deutsche-impf-gegner-100: Unable to download XML: HTTP Error 404: Not Found (caused by <HTTPError 404: 'Not Found'>); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using -U (caused by <HTTPError 404: 'Not Found'>); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using -U
  File "yt_dlp\extractor\common.py", line 730, in _request_webpage
  File "yt_dlp\YoutubeDL.py", line 3558, in urlopen
  File "urllib\request.py", line 531, in open
  File "urllib\request.py", line 640, in http_response
  File "urllib\request.py", line 569, in error
  File "urllib\request.py", line 502, in _call_chain
  File "urllib\request.py", line 649, in http_error_default
```
I also have to use that "--no-check-certificates" command in case of memory Error when trying to download some video from HLS playlist using --downloader aria2c.Errors are like this....
```
02/04 00:15:28 [ERROR] CUID#53 - Download aborted. URI=https://.....
Exception: [AbstractCommand.cc:351] errorCode=1 URI=https://....
  -> [SocketCore.cc:1018] errorCode=1 SSL/TLS handshake failure: Error: Die Sperrfunktion konnte die Sperrung nicht überprüfen, da der Sperrserver offline war.
(80092013)
```
...and when just adding the --no-check-certificates command it works without any issues.

Other problem I get is called...
```
WARNING:←[0m [generic] Could not send HEAD request to https://....: HTTP Error 403: Forbidden
because of missing Origin in request header

WARNING:←[0m [generic] Could not send HEAD request to https://....: HTTP Error 404: Not Found
this error I get after using right Origin paramter
```
The question is why it throws such 404 error after giving yt-dlp all needed paramters?When doing manually or using Curl or any tools they do work.I also don't see any yt-dlp paramter to print out the entire debug infos.Just using -v command or -vU but I don't get much infos just they I did post above.I don't get any infos about the request header paramters which was send + header response.How to get that?

--- a/yt_dlp/extractor/_extractors.py
+++ b/yt_dlp/extractor/_extractors.py
@@ -106,7 +106,6 @@ from .archiveorg import (
 )
 from .arcpublishing import ArcPublishingIE
 from .ard import (
-    ARDIE,
     ARDAudiothekIE,
     ARDAudiothekPlaylistIE,
     ARDBetaMediathekIE,

--- a/yt_dlp/extractor/ard.py
+++ b/yt_dlp/extractor/ard.py
@@ -11,15 +11,12 @@ from ..utils import (
     join_nonempty,
     jwt_decode_hs256,
     make_archive_id,
-    parse_duration,
     parse_iso8601,
     remove_start,
     str_or_none,
-    unified_strdate,
     update_url,
     update_url_query,
     url_or_none,
-    xpath_text,
 )
 from ..utils.traversal import traverse_obj, value
 
@@ -117,118 +114,6 @@ class ARDMediathekBaseIE(InfoExtractor):
                             f['vcodec'] = 'none'
                         formats.append(f)
         return formats
-
-
-class ARDIE(InfoExtractor):
-    _VALID_URL = r'(?P<mainurl>https?://(?:www\.)?daserste\.de/(?:[^/?#&]+/)+(?P<id>[^/?#&]+))\.html'
-    _TESTS = [{
-        # available till 7.12.2023
-        'url': 'https://www.daserste.de/information/talk/maischberger/videos/maischberger-video-424.html',
-        'md5': '94812e6438488fb923c361a44469614b',
-        'info_dict': {
-            'id': 'maischberger-video-424',
-            'display_id': 'maischberger-video-424',
-            'ext': 'mp4',
-            'duration': 4452.0,
-            'title': 'maischberger am 07.12.2022',
-            'upload_date': '20221207',
-            'thumbnail': r're:^https?://.*\.jpg$',
-        },
-    }, {
-        'url': 'https://www.daserste.de/information/politik-weltgeschehen/morgenmagazin/videosextern/dominik-kahun-aus-der-nhl-direkt-zur-weltmeisterschaft-100.html',
-        'only_matching': True,
-    }, {
-        'url': 'https://www.daserste.de/information/nachrichten-wetter/tagesthemen/videosextern/tagesthemen-17736.html',
-        'only_matching': True,
-    }, {
-        'url': 'https://www.daserste.de/unterhaltung/serie/in-aller-freundschaft-die-jungen-aerzte/videos/diversity-tag-sanam-afrashteh100.html',
-        'only_matching': True,
-    }, {
-        'url': 'http://www.daserste.de/information/reportage-dokumentation/dokus/videos/die-story-im-ersten-mission-unter-falscher-flagge-100.html',
-        'only_matching': True,
-    }, {
-        'url': 'https://www.daserste.de/unterhaltung/serie/in-aller-freundschaft-die-jungen-aerzte/Drehpause-100.html',
-        'only_matching': True,
-    }, {
-        'url': 'https://www.daserste.de/unterhaltung/film/filmmittwoch-im-ersten/videos/making-ofwendezeit-video-100.html',
-        'only_matching': True,
-    }]
-
-    def _real_extract(self, url):
-        mobj = self._match_valid_url(url)
-        display_id = mobj.group('id')
-
-        player_url = mobj.group('mainurl') + '~playerXml.xml'
-        doc = self._download_xml(player_url, display_id)
-        video_node = doc.find('./video')
-        upload_date = unified_strdate(xpath_text(
-            video_node, './broadcastDate'))
-        thumbnail = xpath_text(video_node, './/teaserImage//variant/url')
-
-        formats = []
-        for a in video_node.findall('.//asset'):
-            file_name = xpath_text(a, './fileName', default=None)
-            if not file_name:
-                continue
-            format_type = a.attrib.get('type')
-            format_url = url_or_none(file_name)
-            if format_url:
-                ext = determine_ext(file_name)
-                if ext == 'm3u8':
-                    formats.extend(self._extract_m3u8_formats(
-                        format_url, display_id, 'mp4', entry_protocol='m3u8_native',
-                        m3u8_id=format_type or 'hls', fatal=False))
-                    continue
-                elif ext == 'f4m':
-                    formats.extend(self._extract_f4m_formats(
-                        update_url_query(format_url, {'hdcore': '3.7.0'}),
-                        display_id, f4m_id=format_type or 'hds', fatal=False))
-                    continue
-            f = {
-                'format_id': format_type,
-                'width': int_or_none(xpath_text(a, './frameWidth')),
-                'height': int_or_none(xpath_text(a, './frameHeight')),
-                'vbr': int_or_none(xpath_text(a, './bitrateVideo')),
-                'abr': int_or_none(xpath_text(a, './bitrateAudio')),
-                'vcodec': xpath_text(a, './codecVideo'),
-                'tbr': int_or_none(xpath_text(a, './totalBitrate')),
-            }
-            server_prefix = xpath_text(a, './serverPrefix', default=None)
-            if server_prefix:
-                f.update({
-                    'url': server_prefix,
-                    'playpath': file_name,
-                })
-            else:
-                if not format_url:
-                    continue
-                f['url'] = format_url
-            formats.append(f)
-
-        _SUB_FORMATS = (
-            ('./dataTimedText', 'ttml'),
-            ('./dataTimedTextNoOffset', 'ttml'),
-            ('./dataTimedTextVtt', 'vtt'),
-        )
-
-        subtitles = {}
-        for subsel, subext in _SUB_FORMATS:
-            for node in video_node.findall(subsel):
-                subtitles.setdefault('de', []).append({
-                    'url': node.attrib['url'],
-                    'ext': subext,
-                })
-
-        return {
-            'id': xpath_text(video_node, './videoId', default=display_id),
-            'formats': formats,
-            'subtitles': subtitles,
-            'display_id': display_id,
-            'title': video_node.find('./title').text,
-            'duration': parse_duration(video_node.find('./duration').text),
-            'upload_date': upload_date,
-            'thumbnail': thumbnail,
-        }
 
 
 class ARDBetaMediathekIE(InfoExtractor):

